cmd/traffic_layout/info.cc

```diff
@@ -29,22 +29,23 @@
 // Produce output about compile time features, useful for checking how things were built, as well
 // as for our TSQA test harness.
 static void
-print_feature(const char *name, int value, bool json, bool last = false)
+print_feature(ts::string_view const &name, int value, bool json, bool last = false)
 {
   if (json) {
-    printf("    \"%s\": %d%s", name, value, last ? "\n" : ",\n");
+    printf("    \"%.*s\": %d%s", static_cast<int>(name.size()), name.data(), value, last ? "\n" : ",\n");
```

Comment:
Also, it looks really unfortunate that we have to cast here? Is the return type of string_view::size() really not compatible with the "size" that .* expects?

Reply:
The string_view::size() is std::size_t